### ISSUE TITLE
Release 1.8.0 missing code

### ISSUE THREAD

**Issue by jdlm-stripe** [flowtype/flow-for-vscode#424]
About an hour ago it looks like versions 1.7.0 and 1.8.0 were released but they appear to be missing the `build` dir with code. I suspect it was just a failed publish or something like that. 1.5.0 seems like the last good version.

**Comment by vrama628** [comment]
Hi, thanks for raising this. My mistake, looking into it

**Comment by vrama628** [comment]
I just published 1.9.0 to fix this, let me know if it works properly for you

**Comment by Brianzchen** [comment]
I was just on v1.9.0 and there's still and issue, flow isn't able to start on vscode and there's nothing in the status bar now

**Comment by noppa** [comment]
@Brianzchen Weird, 1.9.0 works for me

**Comment by vrama628** [comment]
@Brianzchen to help me determine what the issue may be, could you let me know what output you see when you set `Settings->Flow Configuration->Flow: Log Level` to `trace` then open `View->Output` and select `Flow` from the dropdown menu on the Output pane, and open VSCode to a Flow project.

**Comment by Brianzchen** [comment]
@vrama628 double checked I'm on 1.9.0 and definitely doesn't work. My issue that I can't even start flow at all on vscode. Notable things on the screenshot,

- version is on 1.9.0
- there is no flow status on bottom
- updated to trace
- flow is not in the dropdown
- I can't even manage to start flow or run any commands

<img width="1745" alt="Screen Shot 2021-07-01 at 8 32 43 am" src="https://user-images.githubusercontent.com/12436524/124039820-ffda6200-da46-11eb-91b3-38147f690fe0.png">

Also if it makes and difference I'm running flow 0.154.0

**Comment by jdlm-stripe** [comment]
1.9.0 is working for us now but I saw a weird issue that magically went away for me today. I think the bundled version of Node that comes with VSCode wasn't new enough to have a new addition of [`fs/promises`](https://github.com/flowtype/flow-for-vscode/blob/2b627791c2600814622e664ceb166cb7df24339e/lib/flowLSP/utils/getVerifiedFlowBinPath.js#L6) which was added in 1.9.0. I suspect folks might be running into the same issue.

Yesterday when I was debugging the issue I was seeing node 12.x.x throwing an error while trying to `require('fs/promises')` but for some reason today node 14.x.x is bundled with my vscode. I must have just gotten a background update that fixed it.

TL;DR: try updating VSCode and this extension to 1.9.0. I'm going to close this issue since it appears fixed.

**Comment by Brianzchen** [comment]
Yes just updated my vscode and it works! Thanks @jdlm-stripe

**Comment by mroch** [comment]
https://github.com/flowtype/flow-for-vscode/commit/00a4ff0171b751f8c000541dc352d1b28ba68470 (released as 1.9.1) should restore compatibility with older versions of VS Code. nice find!
